cli/commands/migrate_create.go

```diff
@@ -43,9 +47,13 @@ func newMigrateCreateCmd(ec *cli.ExecutionContext) *cobra.Command {
 		Long:         "Create sql and yaml files required for a migration",
 		Example:      migrateCreateCmdExamples,
 		SilenceUsage: true,
-		Args:         cobra.ExactArgs(1),
+		Args:         cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			opts.name = args[0]
+			if len(args) > 0 {
```

Comment:
I have a feeling that, expecting and enforcing a meaningful name from the user for migration would be better.

Reply:
Okay then I'll leave it as it previously was?

Reply:
Yup, that'll do